### DISCUSSION TITLE
<table> - button filter

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1677] [Q&A]
Hi everyone, I'm using the <Table> control, I would like to know if there is the possibility of moving the filter button to the left side of the column, by default it appears on the right side, and sometimes I can't see it unless I hide it some other column.

thanks Paul

**Comment by r100-stack** [comment]
While I don't believe we offer a way to customize the Filter icon or its location using a prop, you can change the position with CSS styles. So something like:

```scss
.my-column-class > div:first-child {
  justify-content: flex-start; // previously was flex-end
}
```

assuming you passed `my-column-class` as the column class in the column object:

```tsx
const columns = [
  {
    ...
    columnClassName: 'my-column-class'
  },
  ...
];
```

Although this would likely work, I believe this risky. For instance, the selector may need to be modified if we change the dom structure of `iui-table-header-actions-container` in the future.

@iTwin/itwinui-1 @iTwin/itwinui-2 Any ideas? Possibly a safer CSS selector?

**Reply by mayank99** [reply]
i would very much question putting the filter button the left. if it is getting hidden behind other columns, you may have too many columns.

but if you must, then a custom `Header` could potentially work too
